src/main/java/core/basesyntax/files/FileWriterImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.files;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class FileWriterImpl implements FileWriter {
+    private static final String SEPARATOR = System.lineSeparator();
+    private static final String WORD_SEPARATOR = ",";
+
+    @Override
+    public void createReport(Map<String, Integer> report, String path) {
```

Comment:
to follow SRP it's better to move creation of report from this class. This way it'll be responsible only for writing formed data to file